api/routes.py

```diff
@@ -48,6 +48,22 @@ async def subscribe(zone: str, subscriber: Subscriber, session: Session = Depend
             detail="Email already subscribed to this or different zone",
         )
 
+#GET endpoint: function to get subscribers email by zone
+@router.get("/emails/zone/{zone}")
```

Comment:
This should not be an endpoint! We do not want anyone to be able to get all the emails registered for a specific zone. Remove the router.get and keep it as a helper function